docs/tutorials/cpp.md

```diff
@@ -12,20 +12,19 @@ _We are still working on improving the user experience of using ICU4X from other
 
 ## Building and linking ICU4X
 
- - While in our examples you will see invocations like `cargo build -p icu_capi_staticlib --features foo`, neither `cargo install` nor `cargo build` work for non-local library crates (i.e. crates from `crates.io`). To work around this: 
-    - Create a new cargo project (`mkdir icu; cd icu; cargo init`)
-    - Add `icu_capi_staticlib` as a dependency (`cargo add icu_capi_staticlib@1.0.0`)
-    - Specify features with `cargo add icu_capi_staticlib@1.0.0 --features ...`:
+- `cargo build` only supports local crates, so you have to clone the crate using `cargo clone icu_capi_staticlib@1.0.0`
```

Comment:
I'd also suggest updating the stuff below to recommend doing the same for capi headers

Reply:
A user might not know how to get a crate off crates.io, and cargo-clone is more straightforward than wrapping the wrapper in yet another crate.

There's less room for error (version mismatch) if we locate the actual `icu_capi` version that has been built.

Reply:
I understand, but I think the common case is actually that they already have some vendoring system and will want to work within that. If we point people to `cargo clone` it's not immediately obvious what fancy stuff cargo-clone will do that their system won't.

I think the instruction we should give is to get the crate off of crates.io at a particular version (says what they _actually need_ to do), and then specify cargo-clone as _a_ way of doing it (gives them an easy way to do it).

My experience with FFI setups is that everyone's a special snowflake and I find overly specific instructions to be annoying because if I need to do something different than what's listed in the doc, I need to figure out how to unentangle what the app _actually needs_ from what it thinks is the best way of doing so.

Reply:
ok